### PR TITLE
crevado.com template

### PR DESCRIPTION
**Pull request by daflip**
initial template commit

### CODE REVIEW

**Comment by pawel-kow** [comment]
Hi,
The CNAME record with variable-driven host and target is dangerous and opens door for phishing.
Signed requests or phishing warning would be needed in this case.

However according to your help page it's "www" host you want to CNAME, correct?
https://crevado.com/blog/setup-portfolio-custom-domain-name-generic-basic-instructions.html

It would be already better to have "www" static, not as variable in this case.

File: crevado.com.portfolio.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   "records":[
+      {
+        "type": "A",
+        "host": "@",
+        "pointsTo": "%IP%",
+        "ttl": 3600
+       },
+      {
+        "type": "CNAME",
+        "host": "%HOST%",
```
Comment (pawel-kow) [inline crevado.com.portfolio.json]: Needs either static "www", or template marked as `warnPhishing` or signed requests enabled.

Comment (daflip) [reply crevado.com.portfolio.json]: using a static www host now :+1:

**Comment by pawel-kow** [comment]
+1 from me @arnoldblinn pls take a look

**Comment by pawel-kow** [comment]
@daflip for testing/enabling your template IONOS please reach out domain_connect_admin@domain.ionos.com

**Pull request merged**